### DISCUSSION TITLE
¿Qué son los "Requerimientos Funcionales y criterios de aceptación y calidad"?

### DISCUSSION THREAD

**Discussion by Vitaliquid** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#29] [Q&A]
Hola a todos!

Leyendo el contenido y viendo los videos me encanta la idea de poder aportar a la comunidad y ser útil a la organización.
Revisé el archivo requerimientos.doc de la práctica 01-INTRO de la misión front end y en lo particular me encuentro con más dudas que respuestas. por ejemplo me fue suficiente hacer una búsqueda en google, encontré esto sobre el tema de los "requerimientos funcionales y criterios de aceptación de calidad".

https://tales-of-agile-adoption.blogspot.com/2017/06/requerimientos-criterios-de-aceptacion.html

**Comment by CesarEntzana** [comment]
Que buen material, se agradece el aporte es de gran utilidad.
